### ISSUE TITLE
Promotional-"see guild lines" is this a link? also Bold text "must"

### ISSUE THREAD

**Issue by Tayzarcarr** [Tayzarcarr/d1095#4]
"see guildlines" link? also Bold "must"
![image](https://cloud.githubusercontent.com/assets/13252937/15612504/f85279ae-2426-11e6-9389-84d5c8028cf6.png)

**Comment by billyfrost** [comment]
Yes, links are yet to be supplied - Emma needs to ask them for them after next review round
